myth

```diff
@@ -400,67 +357,38 @@ elif (args.graph) or (args.fire_lasers):
             exitWithError(args.outform, "Error saving graph: " + str(e))
 
     else:
-
-        _issues = []
-        report = Report(args.verbose_report)
```

Comment:
I delayed the `report` creation in later place where we need to generate the report